platform/jobs/edxPlatformBokChoyPr.groovy

```diff
@@ -0,0 +1,147 @@
+
+    /* Test secret contains all necessary keys for this job */
+    /* TODO: Use/Build a more robust test framework for this */
+    assert jobConfig.containsKey('open')
+    assert jobConfig.containsKey('jobName')
+    assert jobConfig.containsKey('testengUrl')
+    assert jobConfig.containsKey('platformUrl')
+    assert jobConfig.containsKey('testengCredential')
+    assert jobConfig.containsKey('platformCredential')
+    assert jobConfig.containsKey('platformCloneReference')
+    assert jobConfig.containsKey('admin')
+    assert jobConfig.containsKey('userWhiteList')
+    assert jobConfig.containsKey('orgWhiteList')
+
+    buildFlowJob(jobConfig['jobName']) {
+
+        /* For non-open jobs, enable project based security */
+        if (!jobConfig['open'].toBoolean()) {
+            authorization {
+                blocksInheritance(true)
+                permissionAll('edx')
+            }
+        }
+
+        parameters {
+            stringParam(stringParams.name, stringParams.default, stringParams.description)
+        }
+        logRotator JENKINS_PUBLIC_LOG_ROTATOR() //Discard build after a certain amount of time
+        concurrentBuild() //concurrent builds can happen
+        label('flow-worker-bokchoy') //restrict to jenkins-worker
+        checkoutRetryCount(5)
+        multiscm {
+            git { //using git on the branch and url, clean before checkout
+                remote {
+                        github(jobConfig['testengUrl'])
+                    if (!jobConfig['open'].toBoolean()) {
+                        credentials(jobConfig['testengCredential'])
+                    }
+                }
+                branch('*/master')
+                browser()
+                extensions {
+                    relativeTargetDirectory('testeng-ci')
+                    cleanBeforeCheckout()
+                }
+            }
+            git { //using git on the branch and url, clone, clean before checkout
+                remote {
+                    github(jobConfig['platformUrl'])
+                    refspec('+refs/pull/*:refs/remotes/origin/pr/*')
+                    if (!jobConfig['open'].toBoolean()) {
+                        credentials(jobConfig['platformCredential'])
+                    }
+                }
+                branch('\${ghprbActualCommit}')
+                browser()
+                extensions {
+                    cloneOptions {
+                        reference("\$HOME/" + jobConfig['platformCloneReference'])
+                        timeout(10)
+                    }
+                    cleanBeforeCheckout()
+                    relativeTargetDirectory('edx-platform')
+                }
+            }
+        }
+        triggers { //trigger when pull request is created
+            pullRequest {
+                admins(jobConfig['admin'])
+                useGitHubHooks()
+                triggerPhrase('jenkins run bokchoy')
+                cron('H/5 * * * *')
```

Comment:
I see in the current job configuration that it's set up with a Crontab line, but I'm not sure if/why it's needed. Are we sure that it is? If so please add a comment explaining why.